CMakeLists.txt

```diff
@@ -7,10 +7,13 @@ enable_language(C)
 enable_language(CXX)
 enable_language(ASM)
 
-# What compilers are we using?
-set(CMAKE_C_COMPILER "clang-3.8")
-set(CMAKE_CXX_COMPILER "clang++-3.8")
-set(CMAKE_ASM_COMPILER "clang-3.8")
+if (NOT ${CMAKE_C_COMPILER} MATCHES ".*clang")
```

Comment:
I'd recommend instead trying this test: https://stackoverflow.com/questions/10046114/in-cmake-how-can-i-test-if-the-compiler-is-clang

Reply:
Done in 7fb8fe9.